### DISCUSSION TITLE
Unable to locate a valid CA certificate file

### DISCUSSION THREAD

**Discussion by alkrenawizid** [markshust/docker-magento#657] [Q&A]
Hi,
I try to install automated setup (New Project) then get the below error,

> Unable to locate a valid CA certificate file. You must set a valid 'cafile' option.
A valid CA certificate file is required for SSL/TLS protection.
You can disable this error, at your own risk, by setting the 'disable-tls' option to true.
Creating a "magento/project-community-edition=2.4.3-p1" project at "./"
Unable to locate a valid CA certificate file. You must set a valid 'cafile' option.
A valid CA certificate file is required for SSL/TLS protection.
You can disable this error, at your own risk, by setting the 'disable-tls' option to true.

>   [Composer\Downloader\TransportException]                   
>   The configured cafile was not valid or could not be read.  

![Screen Shot 2022-03-08 at 12 40 00](https://user-images.githubusercontent.com/11616766/157222188-a9a2c825-640c-431b-983a-bea83a8da31d.png)

**Comment by markshust** [comment]
Hi @alkrenawizid, your first command is failing directly after when the "mv rename" lines are displayed. Those shouldn't be displayed or happen, otherwise the entire script may fail. Before attempting to run the `curl` command, that entire directory must be completely empty. You can run `bin/stop` then `bin/removeall`, remove everything in that directory (including hidden files) and try again.

**Reply by alkrenawizid** [reply]
Hi @markshust  👋,
Thanks for your response, after few days it solved via the below command
`composer config --global --unset cafile`

Thanks.
